extensions/wikia/DesignSystem/DesignSystemHelper.class.php

```diff
@@ -13,7 +13,10 @@ class DesignSystemHelper {
 			'sitename',
 			'vertical',
 			'license'
-		]
+		],
+		'global-navigation-search-placeholder-in-wiki' => [
+			'sitename',
+		],
```

Comment:
We do, in the current state of code:
```
if ( !array_key_exists( $fields['key'], static::MESSAGE_PARAMS_ORDER ) ) {
```

And I don't think we should change it. IMO it's better to be explicit like this.